pkg/cosign/tuf/client.go

```diff
@@ -91,16 +98,32 @@ func GetRootStatus(ctx context.Context) (*RootStatus, error) {
 	return t.getRootStatus()
 }
 
+func getMetadataStatus(b []byte) (*MetadataStatus, error) {
```

Comment:
nit: Can we group private functions together, especially since these are all related for getting debugging info?

Reply:
done